source/_integrations/cover.mqtt.markdown

```diff
@@ -421,7 +421,67 @@ cover:
 ```
 
 {% endraw %}
+### Full configuration using advanced templating
 
+The example below shows a full example of how to setup a venetian blind which has a combined position and tilt topic. The blind in the example has moveable slats which tilt with a position change. In the example it takes the blind 6% of the movement for a full rotation of the slats.
+
+Following variable might be used in `position_template`, `set_position_template`, `tilt_command_template` and `tilt_status_template`, `json_attributes_template` (only `entity_id`).
+
+ * `entity_id` - The ID of the entity itself. It can be used to reference its attributes with the help of the [states](/docs/configuration/templating/#states) template function.
+ * `position_open`
+ * `position_closed`
+ * `tilt_min`
+ * `tilt_max`
+
+{% raw %}
+
+```yaml
+# Example configuration.yaml entry
+cover:
+  - platform: mqtt
+    name: "MQTT Cover"
+    command_topic: "home-assistant/cover/set"
+    state_topic: "home-assistant/cover/state"
+    position_topic: "home-assistant/cover/position"
+    set_position_topic: "home-assistant/cover/position/set"
+    tilt_command_topic: "home-assistant/cover/position/set" # same as `set_position_topic`
+    qos: 1
+    retain: false
+    payload_open:  "open"
+    payload_close: "close"
+    payload_stop:  "stop"
+    state_opening: "open"
+    state_closing: "close"
+    state_stopped: "stop"
+    position_open: 100
+    position_closed: 0
+    tilt_min: 0
+    tilt_max: 6
+    tilt_opened_value: 3
+    tilt_closed_value: 0
+    optimistic: false
+    position_template: |-
```

Comment:
We should come up with a simpler example, this is really hard to read and understand for the average user.

Reply:
I agree but it is an example which is a real use case, we can add a simpler one so we have two choices:
- replace this one with a simpler example which just shows how to use `entity_id` (and maybe the place for this example is in the forums)
- add a simple example and still leave this one

Anyhow as I see it we should implement `entity_id` and variables for all MQTT components so there should be many examples for this.

Reply:
I will try to come up wit h a simple example. 
@frenck Please let me know if the complex one shall be kept, or if the usage of variables in templates shall be documented elsewhere? Maybe a description in the templates' documentation would be fine?